CommunityFridgeMapApi/functions/email_service/v1/app.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import json
+
+from send_email import sendEmail
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+recipient = "info@collectivefocus.site"
+
+
+class ContactUs:
+    @staticmethod
+    def lambda_handler(event: dict) -> dict:
+        body = event.get("body", {})
+        if not body:
+            return {"statusCode": 400}
```

Comment:
similar comment as the other ones, change response format

"statusCode": 400,
"headers": {
"Content-Type": "application/json",
"Access-Control-Allow-Origin": "*",
},
"body": {"message": MESSAGE},
}